src/pages/form-validation/password/index.tsx

```diff
@@ -0,0 +1,435 @@
+                                                onClick={(): void => setShowNewPassword(!showNewPassword)}
+                                            >
+                                                {showNewPassword && <Visibility />}
+                                                {!showNewPassword && <VisibilityOff />}
+                                            </IconButton>
+                                        </InputAdornment>
+                                    ),
+                                }}
+                            />
+
+                            <List disablePadding component={'ul'} style={{ marginTop: 8 }}>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.minLengthRequired)}
+                                    <Typography variant={'caption'}>At least 8 characters in length</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1NumberRequired)}
+                                    <Typography variant={'caption'}>At least 1 digit</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1UpperCharRequired)}
+                                    <Typography variant={'caption'}>At least 1 uppercase letter</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1LowerCharRequired)}
+                                    <Typography variant={'caption'}>At least 1 lowercase letter</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1SplCharRequired)}
+                                    <Typography variant={'caption'}>
+                                        At least 1 special character: (valid: ! @ # $ ^ &)
+                                    </Typography>
+                                </ListItem>
+                            </List>
+
+                            <TextField
+                                className={classes.newPasswordInputField}
+                                id={'confirmPassword'}
+                                label={'Confirm Password'}
+                                type={showConfirmPassword ? 'text' : 'password'}
+                                helperText={confirmPasswordError}
+                                onChange={onConfirmPasswordChange}
```

Comment:
Verification should be done on change here (error message should clear as soon as the passwords match) - refer to angular implementation.